proposesolution/gemini.go

```
@@ -0,0 +1,36 @@
+package proposesolution
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/google/generative-ai-go/genai"
+	"github.com/securego/gosec/v2/issue"
+	"google.golang.org/api/option"
+)
+
+func GetSolutionFromGemini(apiKey string, issues []*issue.Issue) {
+	if len(issues) == 0 || apiKey == "" {
+		log.Printf("No issues to solve OR API key is empty")
+		return
+	}
+
+	ctx := context.Background()
+	client, err := genai.NewClient(ctx, option.WithAPIKey(apiKey))
+	if err != nil {
+		log.Fatal(err)
```

I would return the error instead of logging it here, and handle it outside.